logicrunner/common/transcript.go

```diff
@@ -60,3 +98,40 @@ func (t *Transcript) HasOutgoingRequest(
 	}
 	return nil
 }
+
+func contextFromServiceData(data payload.ServiceData) context.Context {
```

Comment:
see common.ContextWithServiceData and FreshContextFromContext

Reply:
no such methods

Reply:
then it is copypasted from package logicrunner. In my PR i moved them to common.

Reply:
which PR? what do you suggest me to do here? I made sure that nobody else uses these functions? should I wait for you PR to be merged or rebase on top of it?

Reply:
my bad, missed new NewTranscriptCloneContext